collection.go

```diff
@@ -0,0 +1,11 @@
+package vt
+
+func PostCollectionFromRawText(client *Client, name, text string) (*Object, error) {
+	obj := NewObject("collection")
+	obj.setMeta("raw", text)
```

Comment:
Are we sending the collection data as `raw` field in the `meta` section? That's a weird behaviour for the API I think.

Reply:
We have two ways for adding items to collections. One is to build relationship descriptors (e.g. `[{"type": "domain", "id": "google.com"}, ...]`) for each one of the entities. The other one is providing a raw text, so the API extracts all the possible IoCs from the text and creates the collection from it. This is useful when for example users copy and paste IoCs from a blogpost's report. We looked at where to send this raw text and we thought the meta information was the most appropriate place (https://jsonapi.org/format/#document-meta)

Reply:
Yes, I think the adding IoCs in raw mode is a useful feature, my concern is about the way in which this done. IMHO that should be done by posting the raw data as an array of strings, like:

```json
{
  "data": [
      "www.foo.com",
      "google.com",
       ....
   ]
}
```

The handler can check if it's receiving a list of object descriptors, or if it's receiving a list of IoC in raw form. The "raw" field in meta doesn't feel right, for example it's weird the field "raw" is submitted in the POST request, but it's not present in the response when you ask for a collection. Also the name is misleading because the IoCs are not actually "metadata" about the collection, it's the collection's data itself.